tenzir/integration/integration.py

```diff
@@ -258,7 +258,7 @@ def run_step(
                 baseline_dir.mkdir(parents=True)
         else:
             LOGGER.debug("comparing step output to baseline")
-        with open(stdout) as out_handle:
+        with open(stdout, "rb") as out_handle:
```

Comment:
Does this also need to be adjusted for the bats-based "functional tests?"

Reply:
cc @tobim

Reply:
Feel free to also just put this into a follow-up issue.

Reply:
It looks like it should work to me. And if it does not, it will most likely tell us about the invalid UTF-8. Hence, I think this is fine without a potentially no-op issue.